<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DevOnOff</class>
 <widget class="QWidget" name="DevOnOff">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>775</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="10" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>测试PDU修改下临界电压OID：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>测试PDU修改下限电压OID：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="QSpinBox" name="delayCloseBox">
     <property name="value">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>延时读取时间：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>延时后开启时间：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="8" column="2" colspan="2">
    <widget class="QLineEdit" name="testIPEdit"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>开关PDU IP地址：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QLineEdit" name="swOidEdit"/>
   </item>
   <item row="9" column="2" colspan="2">
    <widget class="QLineEdit" name="readVolEdit"/>
   </item>
   <item row="11" column="2" colspan="2">
    <widget class="QLineEdit" name="writeMinVolEdit">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>测试PDU读取电压OID：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>延时后关闭时间：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QLineEdit" name="swIPEdit"/>
   </item>
   <item row="10" column="2" colspan="2">
    <widget class="QLineEdit" name="writeCrMinVolEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>开关PDU开关端口OID：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>测试PDU IP地址：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="12" column="0" colspan="4">
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>250</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>250</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="5" column="2" colspan="2">
    <widget class="QSpinBox" name="delayBox">
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="2">
    <widget class="QSpinBox" name="delayOpenBox">
     <property name="value">
      <number>5</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
